styles/myStory.css

```diff
@@ -0,0 +1,65 @@
+.overlay-container {
+  width: 100%;
+  height: 100vh;
+  position: relative;
+  background-image: url("/images/image8.jpg");
+  background-size: cover;
+}
+
+.overlay-text {
+  position: absolute;
+  color: white;
+  font-family: var(--paragraph-sans);
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+}
+
+.overlay-text p {
+  margin-bottom: 50px;
+  margin: 0px;
+  text-transform: uppercase;
+  text-align: center;
+}
+.text-header {
+  font-size: 68px;
+}
+.text-description {
+  padding-top: 20px;
+  font-size: 16px;
+}
+.color-overlay {
+  position: absolute;
+  top: 0;
+  right: 0;
+  bottom: 0;
+  left: 0;
+  background-color: rgba(0, 0, 0, 0.3);
+}
+
+.myStory-content-wrapper {
+  background-color: white;
+}
+.myStory-inner-content {
+  padding: 40px 20vw;
+  margin: 0 auto;
+  font-family: var(--paragraph-sans);
+  letter-spacing: 1px;
+}
+.myStory-inner-content p {
+  line-height: 2.2em;
```

Comment:
what is the computed value for 2.2em?

Reply:
35.2px is not a legible value. Some browser might round it up or down unpredictably. Best practices is to specify even value always

Reply:
Ok, I set the value to 2em